profiles/models.py

```diff
@@ -84,10 +86,14 @@ class Profile(models.Model):
         "self", through=LegalRelationship, symmetrical=False
     )
 
+    def save(self, *args, **kwargs):
+        if self.pk is None and self.user is not None:
+            self.first_name = self.user.first_name or self.first_name
```

Comment:
this would override any values passed by the user through an API, right? what was the use case for this?

Reply:
ah, probably for the initial User object creation. hm 🤔 I guess, we have to do this in the parts where we do `Profile.objects.create()`, cause having this in `save()` can cause unwatned overrides.

Reply:
or is it exactly the spec, that no matter what the user says, we check what we received from the IDP via Tunnistamo 😅

Reply:
The use case was to use users name as an initial value for profile's `first_name` and `last_name` fields. If statement is only going to truthy for new objects with related user present.

Reply:
oh, right, it checks for `pk`. 👍 more succinct would maybe be `if not self.pk and self.user:`?